benchmarks/Benchmarks.Trace/SpanBenchmark.cs

```diff
@@ -15,11 +16,17 @@ namespace Benchmarks.Trace
     [SimpleJob(RuntimeMoniker.NetCoreApp31)]
     public class SpanBenchmark
     {
-        private static Tracer _tracer;
+        private static readonly Tracer Tracer;
```

Comment:
Is there a performance reason for this change?

Reply:
None whatsoever. It was just to please Resharper 😛